tasks.rb

```diff
@@ -90,6 +90,7 @@ def create_hpxmls
     'base-dhw-recirc-nocontrol.xml' => 'base.xml',
     'base-dhw-recirc-temperature.xml' => 'base.xml',
     'base-dhw-recirc-timer.xml' => 'base.xml',
+    'base-dhw-recirc-defaults.xml' => 'base-dhw-recirc-demand.xml',
```

Comment:
Hmm, I guess we can't handle this in base-misc-defaults.xml because there can only be a single hot water distribution system and it's standard in that file. 

How about renaming this file to base-misc-defaults2.xml, so that it will be able to more generally accommodate defaulting of multiple different building components?